Sources/_Concurrency/CMakeLists.txt

```
@@ -0,0 +1,19 @@
+#[[
+Copyright © 2018 Saleem Abdulrasool <compnerd@compnerd.org>.
+All Rights Reserved.
+
+SPDX-License-Identifier: BSD-3
+#]]
+
+add_library(swiftConcurrencyShims
```

This needs to be `swift_Concurrency`.  The name of the library is used for the output, and I believe that the library name is expected to be `swift_Concurrency.dll`/`libswift_Concurrency.so`/`libswift_Concurrency.dylib`.
```suggestion
add_library(swift_Concurrency
```